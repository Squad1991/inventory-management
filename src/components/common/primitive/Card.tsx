import { styled } from '@compiled/react';
import Box from '~src/components/common/primitive/Box';

const Card = styled(Box)({
  border: '1px solid #e0e0e0',
  borderRadius: '0.2rem',
  boxShadow: '0 0 10px rgba(0, 0, 0, .1)',
});

export default Card;
