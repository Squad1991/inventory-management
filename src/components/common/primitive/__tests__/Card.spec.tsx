import { render } from '@testing-library/react';
import React from 'react';
import Card from '~src/components/common/primitive/Card';

describe('Card component', () => {
  it('should match the snapshot', () => {
    const { container } = render(<Card>Hello</Card>);
    expect(container).toMatchSnapshot();
  });
});
