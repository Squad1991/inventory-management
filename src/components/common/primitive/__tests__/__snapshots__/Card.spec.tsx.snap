// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Card component > should match the snapshot 1`] = `
<div>
  <style
    data-cmpld="true"
  >
    ._19itl2jy{border:1px solid #e0e0e0}._2rko1rrh{border-radius:.2rem}._16qsd8sd{box-shadow:0 0 10px rgba(0,0,0,.1)}
  </style>
  <style
    data-cmpld="true"
  >
    ._1e0c1txw{display:flex}
  </style>
  <div
    class="_1e0c1txw _19itl2jy _2rko1rrh _16qsd8sd"
  >
    Hello
  </div>
</div>
`;
